manifest/process.go

```diff
@@ -107,14 +108,24 @@ func (p *Process) GenerateArgs(opts *ArgOptions) []string {
 
 	for _, volume := range p.service.Volumes {
 		if !strings.Contains(volume, ":") {
-			home, err := homedir.Dir()
-			if err != nil {
-				log.Fatal(err)
+			home := ""
+			var err error
+
+			switch runtime.GOOS {
+			case "windows":
+				home = "/tmp" // prefix with /tmp to use Docker Volume
```

Comment:
@ddollar changed to /home/convox.

But /tmp/.convox is persisted across `convox starts` too.